Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -289,16 +302,7 @@ public final class IndexStoreDB {
   public func symbolProvider(for sourceFilePath: String) -> SymbolProviderKind? {
```

Comment:
IIRC this API was added for what's now using the provider from the occurrence instead, should we just remove?

Reply:
(https://github.com/apple/indexstore-db/pull/177)

Reply:
I’ll remove it in a follow-up PR.